src/components/header.vue

```diff
@@ -10,8 +11,14 @@ export default {
 <style scoped>
 .header.ui.row {
   text-align: center;
-  background-color: #0980fb !important;
+  background-color: #fff;
```

Comment:
@georgemickleburgh We can simply remove `background-color`, the default color is white so we do not need to define it explicitly